aten/src/ATen/Context.h

```diff
@@ -119,6 +123,8 @@ class CAFFE2_API Context {
   bool enabled_cudnn = true;
   bool deterministic_cudnn = false;
   bool benchmark_cudnn = false;
+  bool enabled_fbgemm = true;
+  bool enabled_qnnpack = false;
```

Comment:
How about we use one enum flag instead of multiple boolean flags like what @dzhulgakov proposed? Users can force the specific engine to use with this flag. When it is not set, pick the engine with some predefined rules, e.g. pick the optimal engine according to HW platform.

https://github.com/pytorch/pytorch/pull/25338#issuecomment-526475302
> have global runtime API for debugging, something like torch.backends.quantized.engine which is None by default, but can be set to FBGEMM or QNNPACK